src/sentry/tasks/sentry_apps.py

```diff
@@ -244,21 +244,25 @@ def notify_sentry_app(event, futures):
 
 
 def send_webhooks(installation, event, **kwargs):
-    project_ids = Project.objects.filter(
-        organization_id=installation.organization_id,
-    ).values_list('id', flat=True)
-
-    servicehooks = ServiceHook.objects.filter(
-        project_id__in=project_ids,
-    )
+    try:
+        servicehook = ServiceHook.objects.get(
+            organization_id=installation.organization_id,
+            actor_id=installation.id,
+        )
+    except ServiceHook.DoesNotExist:
+        return
 
-    for servicehook in filter(lambda s: event in s.events, servicehooks):
-        if not servicehook.created_by_sentry_app:
-            continue
+    if event not in servicehook.events:
+        return
 
-        if servicehook.sentry_app != installation.sentry_app:
-            continue
+    # The service hook applies to all projects if there are no
+    # ServiceHookProject records. Otherwise we want to check if we
+    # should send the webhook or not.
+    project_ids = ServiceHookProject.objects.filter(
+        service_hook_id=servicehook.id,
+    ).values_list('project_id', flat=True)
```

Comment:
We can make this a little more descriptive if we use `exists` and name the variable accordingly. I'm thinking something like

```python
project_specific = ServiceHookProject.objects.filter(service_hook_id=sevicehook.id).exists()

if not project_specific:
    ...
```

That makes the intent very clear, which is what we should be going for with our code. Comments help, but should be a last resort for conveying intent or logic, imo.

Reply:
I think this makes sense given the current context. I was trying to think of what would make it easier going forward when we'll need to know what projects are in that table vs just knowing that there aren't any. BUT since we'll have to revisit anyway (re: marks comment) can just wait til then